q0_gui.py

```diff
@@ -78,6 +82,8 @@ def __init__(self, parent=None, args=None):
 
         self.ui.restore_cryo_button.clicked.connect(self.restore_cryo)
 
+        self.region_selection_items = []
```

Comment:
I think I'm confused about what this list is. It looks like it's shared by both Q0 and calibration measurements? Unless I'm misreading it, I think it would make more sense to have this be inside of the `DataRun` class. Like, one region per run. And then a mutable data buffer can be updated automatically instead of overwriting the run's `ll_data`. I'm thinking maybe rename `ll_data` to `_ll_data` to indicate lack of mutability and then `ll_data` can be the one that gets written to by using a subset of `_ll_data`. If that doesn't make sense, we can chat